alltest.sh

```diff
@@ -4,30 +4,15 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
-test::package() {
-  local package="${1:-notset}"
-
-  if [[ "${package}" == "notset" ]]; then
-    &>2 echo "empty package name"
-    exit 1
-  fi
-
-  local coverage_file="${package//\//.}.cover.out"
-  echo "[TEST] package=${package}, coverage=${coverage_file}"
-  go test -tags "$UNITTESTTAGS" -race -failfast -parallel 16 -cpu 16 -coverpkg="github.com/CovenantSQL/CovenantSQL/..." -coverprofile "${coverage_file}" "${package}"
-}
-
 main() {
   make clean
   make -j6 bp miner observer
 
-  # test package by package
-  for package in $(go list ./... | grep -v "/vendor/"); do
-    test::package "${package}"
-  done
+  go test -tags "$UNITTESTTAGS" -race -failfast -parallel 16 -cpu 16 -coverprofile main.cover.out $(go list ./... | grep -v CovenantSQL/api)
```

Comment:
Shall we exclude `vendor` folders?

Reply:
`vendor` folder is by default excluded by `go list` or `go test` command.